src/installer/tests/HostActivation.Tests/PortableAppActivation.cs

```diff
@@ -458,22 +458,24 @@ public void AppHost_CLI_FrameworkDependent_MissingRuntimeFramework_ErrorReported
             using (new TestArtifact(invalidDotNet))
             {
                 Directory.CreateDirectory(invalidDotNet);
-
-                string expectedErrorCode;
                 string expectedUrlQuery;
-                string expectedUrlParameter = null;
+                string expectedStdErr = null;
```

Comment:
Can you also change the name of this test `ErrorReportedInDialog` -> `ErrorReportedInStdErr`?